main/pipeline/pipeline.cc

```diff
@@ -572,7 +571,7 @@ ast::ParsedFile typecheckOne(core::Context ctx, ast::ParsedFile resolved, const
         }
         CFGCollectorAndTyper collector(opts);
         {
-            result.tree = ast::TreeMap::apply(ctx, collector, move(resolved.tree));
+            ast::TreeMap::apply(ctx, collector, move(resolved.tree));
```

Comment:
(non-blocking) It would be nice if we had a different name here like `sideEffectfulMap` to indicate that we only care about side effects and the `TreeMap` is deliberately not doing any transformations.